### PR TITLE
Rename prop 'shape' to 'corners'

### PR DESCRIPTION
**Pull request by rahilvora**
Rename prop 'shape' to 'corners' for components Box, Mask, and Touchable (including dependent components Avatar, GroupAvatar, Pog, Toast, VideoPlayhead). Also updates the docs

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit e8eab7042e0f368ac368ff61c3c53440b7e87534

https://deploy-preview-301--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  :file_folder: Bundlesize - <i>Bundle size increase detected, please review</i>
  </td>
    </tr>
  </tbody>
</table>


Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
 ---  |  ---  |  ---  |  ---  |  ---  |  ---  |  --- 
gestalt.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 261.37 KB | 261.42 KB | 58.72 KB | 58.72 KB
gestalt.es.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 258.7 KB | 258.75 KB | 58.35 KB | 58.34 KB
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: packages/gestalt/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gestalt",
-  "version": "0.77.0",
+  "version": "0.77.1",
```
Comment (jackhsu978) [inline packages/gestalt/package.json]: I don't think we want to bump up the version until we are doing a release. this should be kept the same.

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,14 @@
 
 </details>
 
+## 0.77.1 (August 2, 2018)
```
Comment (jackhsu978) [inline CHANGELOG.md]: please move this to the unreleased section above

Comment (rahilvora) [reply CHANGELOG.md]: Sure. Will update this.

**Comment by rahilvora** [comment]
@pfarejowicz Hey Peter, I can try to write the codemods for this change. I would like to understand what is the naming (It is some numbers) of the folders of the codemods directory means. Should I create a new folder and write a new file in that?

**Comment by pfarejowicz** [comment]
@rahilvora the folders in that directory are named based on the current version of gestalt and the future version of gestalt that introduces the breaking change. Since we are currently on version 0.77, your change once landed will be part of version 0.78. Feel free to create a new folder with those numbers and then add a file with a descriptive name for the prop codemod 🙂

**Comment by rahilvora** [comment]
@jackhsu978 @pfarejowicz  have made the requested changes. Kindly, let me know if any changes are required. Thank you.

**Comment by chrislloyd** [comment]
Closing out. Feel free to re-open if still relevant.

**Pull request closed without merging**